autoversioner/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
```

Comment:
this config should go in the shared cloudbuild.yaml

Reply:
done